frontend/www/js/omegaup/components/notification/Clarifications.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <li class="dropdown">
+    <a
+      aria-expanded="false"
+      aria-haspopup="true"
+      class="notification-button dropdown-toggle"
+      data-toggle="dropdown"
+      href="#"
+      role="button"
+      v-on:click="unread = false"
+      ><span class="glyphicon glyphicon-bell"></span>
+      <span
+        class="notification-counter label"
+        v-bind:class="{ 'label-danger': unread }"
+        v-if="clarifications &amp;&amp; clarifications.length &gt; 0"
+        v-model="unread"
+        >{{ clarifications.length }}</span
+      ></a
+    >
+    <ul class="dropdown-menu">
+      <li class="empty" v-if="!clarifications || clarifications.length === 0">
+        {{ T.notificationsNoNewNotifications }}
+      </li>
+      <li v-else="">
+        <ul class="notification-drawer">
+          <li v-for="clarification in clarifications">
+            <button
+              v-bind:aria-label="T.wordsClose"
+              class="close"
+              type="button"
+              v-on:click.prevent="onCloseClicked(clarification)"
+            >
+              <span aria-hidden="true">×</span>
+            </button>
+            <a v-bind:href="anchor(clarification)"
+              ><span>{{ clarification.problem_alias }}</span> —
+              <span>{{ clarification.author }}</span>
+              <pre>{{ clarification.message }}</pre>
+              <hr v-if="clarification.answer" />
+              <pre v-if="clarification.answer">{{
+                clarification.answer
+              }}</pre></a
+            >
+          </li>
+        </ul>
+      </li>
+      <li
+        class="divider"
+        role="separator"
+        v-if="clarifications &amp;&amp; clarifications.length &gt; 1"
+      ></li>
+      <li v-if="clarifications &amp;&amp; clarifications.length &gt; 1">
+        <a href="#" v-on:click.prevent="onMarkAllAsRead"
+          ><span class="glyphicon glyphicon-align-right"></span>
+          {{ T.notificationsMarkAllAsRead }}</a
+        >
+      </li>
```

Comment:
aquí funciona

```suggestion
      <template v-if="clarifications && clarifications.length > 1">
        <li
          class="divider"
          role="separator"
        ></li>
        <li>
          <a href="#" v-on:click.prevent="onMarkAllAsRead"
            ><span class="glyphicon glyphicon-align-right"></span>
            {{ T.notificationsMarkAllAsRead }}</a
          >
        </li>
      </template>
```

esto funciona? en general si tienes más de un elemento que se tiene que renderear condicionalmente, hay que intentar usar un `<template>` siempre que se pueda.

Reply:
Entendido